tests/custom_steps.js

```diff
@@ -76,4 +77,55 @@ module.exports = () => actor({
       this.wait(interval);
     }
   },
+
+  /**
+   * Create URL method
+   *
+   * @param url start
+   * @param parameters object
+   * @returns {Promise<void>}
+   *
+   * @example
+   * getDashboardUrlWithParams('http://example.com', { environment: 'ps-dev', from: 'now-1' });
+   */
+  getDashboardUrlWithParams(url, parameters) {
+    const body = {};
+
+    body.from = 'now-5m';
+    body.to = 'now';
+    Object.entries(parameters).forEach(([key, value]) => {
+      switch (key) {
+        case 'environment':
+          body['var-environment'] = value;
+          break;
+        case 'node_name':
+          body['var-node_name'] = value;
+          break;
+        case 'service_name':
+          body['var-service_name'] = value;
+          break;
+        case 'columns':
+          value ? body.columns = value : body.columns = '%5B%22load%22,%22num_queries%22,%22query_time%22%5D';
```

Comment:
why do we need this special list of columns? isn't it a default one?

Reply:
yes, body.columns = '%5B%22load%22,%22num_queries%22,%22query_time%22%5D' - it is default value for columns.

Reply:
we don't need to hardcode it then. It is appended automatically by the app itself

Reply:
ok. removed it